### ISSUE TITLE
There is no data provider registered that can provide view data.

### ISSUE THREAD

**Issue by arhill05** [AngularDoc/angulardoc.github.io#40]
When attempting to use the Copilot extension, VS Code displays the below message:

![image](https://user-images.githubusercontent.com/13209244/56358297-3e5c0d00-61ac-11e9-84ee-cb63e22b7668.png)

Furthermore, clicking refresh or attempting to use a command from the command palette results in an error.

Click refresh, see the following error:
![image](https://user-images.githubusercontent.com/13209244/56358334-52077380-61ac-11e9-8007-2ade8bd4a582.png)

Things I've tried: 
- Reinstalling
- Installing version 1.0.0
- Ensure AngularDoc is not installed
- Reloading window
- Restarting VS Code
- Rebooting

VS Code About Box:
![image](https://user-images.githubusercontent.com/13209244/56358522-d823ba00-61ac-11e9-8e43-012bb7f3f2c0.png)

Extensions installed:
```
- 2gua.rainbow-brackets@0.0.6
- alefragnani.project-manager@10.5.1
- be5invis.vscode-custom-css@3.0.3
- christian-kohler.path-intellisense@1.4.2
- dbaeumer.vscode-eslint@1.8.2
- DotJoshJohnson.xml@2.4.0
- eamodio.gitlens@9.6.2
- eg2.tslint@1.0.43
- esbenp.prettier-vscode@1.8.1
- formulahendry.auto-close-tag@0.5.6
- formulahendry.auto-rename-tag@0.0.15
- k--kato.docomment@0.1.6
- Metatype.copilot-vscode@1.0.1
- mitchdenny.ecdc@1.0.5
- ms-mssql.mssql@1.5.0
- ms-vscode.csharp@1.19.0
- ms-vscode.vscode-typescript-tslint-plugin@1.0.0
- ms-vsliveshare.vsliveshare@1.0.67
- msjsdiag.debugger-for-chrome@4.11.3
- PeterJausovec.vscode-docker@0.6.1
- PKief.material-icon-theme@3.7.0
- pnp.polacode@0.2.2
- pranaygp.vscode-css-peek@2.2.0
- Shan.code-settings-sync@3.2.8
- vscode-icons-team.vscode-icons@8.5.0
- WakaTime.vscode-wakatime@1.3.0
- wayou.vscode-todo-highlight@1.0.4
- wesbos.theme-cobalt2@2.1.6
- xabikos.JavaScriptSnippets@1.7.2
- yummygum.city-lights-theme@1.1.0
- yzhang.markdown-all-in-one@2.2.0
```

**Comment by c0h1b4** [comment]
Same here. VueJs project.

command 'copilot.explorer.refresh' not found

**Comment by danilofuchs** [comment]
Same here for React with typescript.
Seems like a generalized bug.

**Comment by c0h1b4** [comment]
Uninstalling as quick as installing. See you in version 2.0.

Good idea, bad implementation.

**Comment by agiledigits** [comment]
@arhill05 @c0h1b4 @danilofuchs Could one of you do a Skype session with us to help us debug this problem?

**Comment by agiledigits** [comment]
@arhill05 @c0h1b4 @danilofuchs Please try v1.0.2 and see if you still have problems.

**Comment by danilofuchs** [comment]
Fixed in version 1.1.0! Thanks for the quick fix!

(Using React with create-react-app and typescript)

**Comment by ChuckJonas** [comment]
I still get this in 1.1.0.  When the app launches I get an error: `Copilot server:connect ECONNREFUSED 127.0.0.1:63848`
